packages/appium/test/config-specs.js

```diff
@@ -27,11 +60,7 @@ describe('Config', function () {
       // need to be able to write to process.version
       // but also to have access to process methods
       // so copy them over to a writable object
-      let tempProcess = {};
-      for (let [prop, value] of _.toPairs(process)) {
-        tempProcess[prop] = value;
-      }
-      process = tempProcess; // eslint-disable-line no-global-assign
+      process = {...process}; // eslint-disable-line no-global-assign
```

Comment:
not sure I understand what happens here

Reply:
global process is assigned to a shallow copy of the original global process.  essentially just that the line I added did the same thing as the previous 5 or whatever

Reply:
fwiw: this is hard to test because `process.version` is read-only.  that suggests to me that we should not actually be interfacing with it directly, and instead provide, say, a function which wraps `process.version`.  we can then stub that function for test purposes.

Reply:
OTOH... I am not sure if we actually even need to be checking node versions?  I suppose if you miss the engine warnings and install anyway you _might_ see the warning out of Appium, _if_ the syntax is intelligible by the runtime.

Reply:
ping @jlipps  ^